src/libraries/Microsoft.PowerFx.Interpreter/NotDelegableException.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.PowerFx.Interpreter
+{
+    public sealed class NotDelegableException : Exception
```

Comment:
Add a comment that this is used to fail attempted delegation and fall back to in memory evaluation, and should always be caught.

Reply:
added